src/unix/os390-syscalls.c

```diff
@@ -136,11 +136,11 @@ uv__os390_epoll* epoll_create1(int flags) {
   if (lst == -1)
     return NULL;
```

Comment:
Unrelated bug but the lock is not released.  (Also, `lst == -1`?  It's a pointer.)

Question: `flags` is unused even though the call site in os390.c passes `UV__EPOLL_CLOEXEC`.  Bug?

Question: is it necessary to take out the lock _before_ the malloc call?  `lst` is unobservable until it is inserted into the global queue, right?  I think you should be able to rewrite the function to this:
```c
uv__os390_epoll* epoll_create1(int flags) {
   uv__os390_epoll* lst;
 
   lst = uv__malloc(sizeof(*lst));
   if (lst != NULL) {
     lst->size = 0;
     lst->items = NULL;
     uv_once(&once, epoll_init);
     uv_mutex_lock(&global_epoll_lock);
     QUEUE_INSERT_TAIL(&global_epoll_queue, &lst->member);
     uv_mutex_unlock(&global_epoll_lock);
  }

   return lst;
}
```

Reply:
I have removed the `UV__EPOLL_CLOEXEC` arg that is passed on to `epoll_create` because we forcefully CLOEXEC the epoll file descriptor in this implementation anyway.
I will leave the flags parameter in `epoll_create` though to keep it consistent with linux.
There might be flags that we create in the future that could be useful.